src/utils/formatters.ts

```diff
@@ -0,0 +1,50 @@
+import { isArray } from 'lodash'
+import { IMiddleware } from '../types/interfaces'
+
+/**
+ * Works only with first middleware argument and prints his value if it consists of a primitive value array or an object containing such arrays
+ * @param {string} middlewareName
+ * @param {IMiddleware} middleware object containing all necessary information about actual middleware
+ * @return { string } middleware's description
+ * */
+export const basicArrayFormatter = (middlewareName: string, middleware: IMiddleware): string => {
+	const permissionsResult = `${middlewareName}: `
```

Comment:
@kot-1999 this function is no more related to permissions, so all permission keywords should be replaced with some generics, for example `permissionsResult` -> `middlewareResult`